### PR TITLE
Adds hardware change digest notes to beta page

### PR DESCRIPTION
**Pull request by gracecanfield**
https://docs.google.com/document/d/1LL9zs3oz-3tJXtqVU7IRXLRoBn2HrmNgwzdpLy209lc/edit#

### CODE REVIEW

File: 4_Manufacturing_Design/Beta_Build_Instructions/README.md [hunk 1]
```diff
@@ -1,7 +1,23 @@
...
+
+We're planning to move the STM32 (nucleo module) onto the PCB, so there won't be a need to buy nucleo module. 
+
+## STM32 Updates
+
+We're planning to add some kind of Flash memory to support non-volatile storage of configuration parameters. Part selection is still TBD; we'll either be moving to dual flash bank STM32 or add an i2c EEPROM (already planned for PCB Rev A).
+
+## Stepper Drivers Updates
+
+We're planning to move the steppers driver onto the PCB. No need to buy stepper drivers - remove from BOM] Moving stepper drivers onto board - Pizza build version no change (see above “stepper drivers” for details)
```
Comment (martukas) [inline 4_Manufacturing_Design/Beta_Build_Instructions/README.md:19]: stray bracket ] and see above where?

File: 4_Manufacturing_Design/Beta_Build_Instructions/README.md [hunk 2]
```diff
@@ -1,7 +1,23 @@
...
-## Venturi updates
+## Venturi Updates
 
-The plan is for beta to use 3/4"ID tubing throughout. The venturi will most likely be redesigned to have barbs that fit 3/4"ID tubing directly.
\ No newline at end of file
+We're planning to use 3/4"ID tubing throughout. The venturi will most likely be redesigned to have barbs that fit 3/4" ID tubing directly.
+
+## Nucleo Module Updates
+
+We're planning to move the STM32 (nucleo module) onto the PCB, so there won't be a need to buy nucleo module.
```
Comment (martukas) [inline 4_Manufacturing_Design/Beta_Build_Instructions/README.md:11]: I would remove "(nucleo module)"

File: 4_Manufacturing_Design/Beta_Build_Instructions/README.md (hunk 1 above)
Comment (gracecanfield) [reply 4_Manufacturing_Design/Beta_Build_Instructions/README.md]: Fixed.

File: 4_Manufacturing_Design/Beta_Build_Instructions/README.md (hunk 2 above)
Comment (gracecanfield) [reply 4_Manufacturing_Design/Beta_Build_Instructions/README.md]: Updated.

File: 4_Manufacturing_Design/Beta_Build_Instructions/README.md [hunk 3]
```diff
@@ -1,7 +1,23 @@
 # Beta Build Instructions
```
Comment (martukas) [inline 4_Manufacturing_Design/Beta_Build_Instructions/README.md:1]: And at this point they are not so much instructions, but just a plan?
Also, alpha build is linked from main page, but beta is not. Should we add a link?

Comment (gracecanfield) [reply 4_Manufacturing_Design/Beta_Build_Instructions/README.md]: Good point - updated

**Pull request merged**